uint8 id
uint32 system_time
float32 dis
uint8 dis_status
uint16 signal_strength
uint8 range_precision
